src/apps/peer/udpserver.c

```diff
@@ -37,6 +37,12 @@
 #include "stun_buffer.h"
 
 #include <limits.h> // for USHRT_MAX
+#ifdef _WIN32
+#include <winsock2.h>
+#define close closesocket
```

Comment:
check other parts of the code, they use a different pattern:

#if defined(WINDOWS)
/** Do the platform-specific call needed to close a socket returned from
    socket() or accept(). */
#define socket_closesocket(s) closesocket(s)
#else
/** Do the platform-specific call needed to close a socket returned from
    socket() or accept(). */
#define socket_closesocket(s) close(s)
#endif

Also I think socket_closesocket() should be already included in this file so you can use it directly